### PR TITLE
Protect fetch time parameter

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/15432

## Description
validate fetch time param and set default value when is empty

### CODE REVIEW

File: Tests/id_set.json [hunk 1]
```diff
@@ -8892,6 +8892,30 @@
...
+                    "fd-list-agents", 
+                    "fd-create-ticket-note", 
+                    "fd-delete-ticket", 
+                    "fd-search-tickets"
+                ], 
+                "tests": [
+                    "Freshdesk-Test"
+                ]
+            }
+        },
```
Comment (liorblob) [inline Tests/id_set.json]: Why is this here?...

File: Tests/secrets_white_list.json [hunk 2]
```diff
@@ -646,7 +650,11 @@
       "sn1pr11mb0638d849680e1325bc4f7951cc970@sn1pr11mb0638.namprd11.prod.outlook.com",
       "idov@demisto.com",
       "bar.hochman@demisto.com",
-      "powershellteam@hotmail.com"
+      "powershellteam@hotmail.com",
+      "example3@gmail.com",
+      "example1@gmail.com",
+      "jeffrey.collings@gmail.com",
+      "example2@gmail.com"
```
Comment (liorblob) [inline Tests/secrets_white_list.json]: ???

Comment (yardensade) [reply Tests/secrets_white_list.json]: Do not add irrelevant emails that are not supposed to be exposed on public repo please.

Comment (adi88d) [reply Tests/secrets_white_list.json]: done

**Comment by liorblob** [comment]
@anara123 I don't think the error should come from there, but it's always good to check the function inputs

**Comment by anara123** [comment]
function should protect herself, if there are extra space, then we can strip it, if None passed then invalid input = exception.
And in each integration in the beginning of the code, we should run validate_date_range(date_range)

**Comment by yardensade** [comment]
If we are going to strip anyway, we should strip from the function itself, no point validating it over and over again in each integration.
Also might be good to be able and pass into the function a default argument in case you don't want the function to fail on lack of input instead of creating another default var in the code.
So if you don't provide it AND there is no actual input only then it the function will throw exception.

**Comment by anara123** [comment]
there are cases that this kind of protective code could be a correct way, but in this case I think it is not. by allowing default argument it means that user can provide invalid date range like "gkjfdnkhjdfnh" and we will fetch incidents 3 days back instead of returning an error and asking him to fix it

**Comment by liorblob** [comment]
It will only do it in cases where the parameter is an empty space, if it has a concrete value then it will be sent to the function
